src/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -113,9 +113,8 @@ export default function MaxSlippageSelect() {
   }, [focus, slippage, setSlippage])
 
   const processInput = useCallback(
-    (input: string | undefined) => {
-      setMaxSlippageInput(input || '')
-      const max = input ? +input : undefined
+    (max: string | undefined) => {
+      setMaxSlippageInput(max || '')
```

Comment:
this line has always bothered me a little, but are you sure you want to drop the type coercion here?

Reply:
type coercion is no longer necessary, as the coercion will now happen in `toPercent`.
The logic is necessary for smooth UX.